helm/grafana/values.schema.json

```diff
@@ -644,10 +693,10 @@
                             "type": "boolean"
                         },
                         "name": {
-                            "type": ["string", "null"]
+                            "type": "null"
```

Comment:
I think we need to keep those ["string", "null"] right?

Reply:
Fixed